conf/default.yaml

```diff
@@ -9,57 +9,16 @@ server:
   port: "{{ envVars.port }}"
   address: '0.0.0.0'
 routePrefix: "{{ envVars.routePrefix }}"
-sentryFilter:
-  - 'error'
-  - 'warning'
-  - 'user-error'
-  - 'server-error'
-  - 'sentry'
-
 plugins:
   hapi-health:
     token: "{{ envVars.healthToken }}"
     endpoint: "{{routePrefix}}/health"
     auth: false
+  hapi-prom:
+    _enabled: "{{ truthy(envVars.enableProm)}}"
   hapi-logr:
     unhandledRejection: true
     uncaughtException: true
-    reporters:
```

Comment:
we should add a param that we can pass into logr-all to enable certain reporters. basically same as env var, but explicit.